matchms/importing/load_from_msp.py

```diff
@@ -31,6 +33,9 @@ def parse_msp_file(filename: str) -> Generator[dict, None, None]:
 
                 for peak in peak_pairs:
                     mz, intensity = get_peak_values(peak)
+                    comment = get_peak_comment(peak)
+                    if comment is not None:
+                        peak_comments.update({mz: comment})
```

Comment:
I think this could be integrated into getting peak values which could then return a triple